### PR TITLE
Render list from api

### PR DESCRIPTION
**Pull request by ShahierNashaat**
Pull request for reviewing display list of movies from API in the javascript capstone project.

Render list from api.

### CODE REVIEW

File: src/api-controller.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const getMovies = () => fetch('https://api.tvmaze.com/shows')
```
Comment (xtrahuman) [inline src/api-controller.js:1]: I think it will be nice to use async functions and await as advised by Microverse in the previous similar task

**Review by xtrahuman — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED ♻️ 
Hi @ShahierNashaat  👋,

Nice job so far with this project 😄, that was really fast

Required Changes ♻️
  - [ ]  _Check the comments under the review._
Cheers and Happy coding!👏👏👏

File: src/api-controller.js (hunk 1 above)
Comment (ShahierNashaat) [reply src/api-controller.js:1]: I used it in index.js file

**Review by xtrahuman — APPROVED** [review]
Status: Approved ✔️
Hi @ShahierNashaat ,

The footer section is complete🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: 🎉💯🌟

**Pull request merged**